src/main/java/com/zombiecastlerush/building/Castle.java

```diff
@@ -17,13 +17,17 @@ public Castle(){
 
         //add connected rooms to room
         eastWing.addConnectedRooms(castleHall);
-        castleHall.addConnectedRooms(drawBridge);
-        castleHall.addConnectedRooms(eastWing);
-        castleHall.addConnectedRooms(westWing);
-        drawBridge.addConnectedRooms(westWing);
-        drawBridge.addConnectedRooms(castleHall);
-        westWing.addConnectedRooms(castleHall);
-        westWing.addConnectedRooms(drawBridge);
+        castleHall.addConnectedRooms(drawBridge,eastWing,westWing);
+        drawBridge.addConnectedRooms(westWing,castleHall);
+        westWing.addConnectedRooms(castleHall,drawBridge);
+
+        //add Challenge to room
+        eastWing.setChallenge(new Puzzle("East-Wing Puzzle","What is (2+2) X (2-2)?","0"));
+        eastWing.getChallenge().inventory.addItems(new Item("Mr.Big Potato","This is a very big potato"));
+        westWing.setChallenge(new Puzzle("West-Wing Puzzle","What is (2+2) X (2-2)?","0"));
```

Comment:
I would definitely stay away from using multiple word names here and in Item, because our textParser can't handle more than two words right now. Use one single word for Item and Puzzle names et al.